config/rbac/role.yaml

```diff
@@ -669,6 +675,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - features.opendatahub.io
+  resources:
+  - featuretrackers/status
+  verbs:
+  - delete
+  - get
+  - patch
+  - update
```

Comment:
@bartoszmajsak I'm not sure why this is missing. It is being added by `make manifests generate`.

Reply:
yeah, it's been forgotten in the previous PR I believe.